examples/grommet/src/components/PollResume/QuestionResume.tsx

```diff
@@ -0,0 +1,101 @@
+import { useVerna } from '@openfun/verna';
+import { Box, Card, CardBody, Meter, Text } from 'grommet';
+import _ from 'lodash';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { schemaFormType } from '../../data/schemas';
+
+interface QuestionResumeProperties {
+  questionData: unknown;
+  poll: schemaFormType;
+}
+
+export default function QuestionResume({ questionData, poll }: QuestionResumeProperties) {
+  const { isEditor, schemaTranslations } = useVerna();
+  const { formatMessage, locale } = useIntl();
+  const nbAnswers =
+    _.reduce(questionData, (total: number, current: number) => total + current) || 0;
+  const hasAnswered = false;
+  const pollRJSF = poll.schema?.properties?.[Object.keys(poll.schema.properties)[0]];
+
+  return (
+    <Card background="rgba(145 110 208 / 18%)" style={{ marginBottom: '15px', padding: '15px' }}>
+      <CardBody>
+        {pollRJSF?.title && (
+          <legend>
+            <FormattedMessage id={pollRJSF.title} />
+          </legend>
+        )}
+        {pollRJSF?.description && (
+          <p>
+            <FormattedMessage id={pollRJSF.description} />
+            <br />
+          </p>
+        )}
+        <small>{`${nbAnswers} réponses`}</small>
+        {(isEditor || hasAnswered) && (
+          <Box direction="column" gap="10px">
+            {Object.keys(questionData).map((dataName: string, index) => {
+              const label = Object.keys(schemaTranslations[locale] || {}).includes(dataName)
+                ? formatMessage({ id: dataName })
+                : dataName;
+              const value = questionData[label];
+              const percentage = parseFloat(
+                Number(nbAnswers ? (value / nbAnswers) * 100 : 0).toFixed(1),
+              );
+
+              return (
+                <Box
+                  key={`${index}-${dataName}`}
+                  direction="row"
+                  style={{ alignItems: 'center', gap: '10px' }}
+                >
+                  <div style={{ alignSelf: 'flex-start', position: 'relative' }}>
+                    <div style={{ position: 'absolute' }}>
+                      <Box
+                        align="center"
+                        direction="row"
+                        style={{ marginLeft: '20px', marginTop: '8px' }}
```

Comment:
```suggestion
                        style={{ marginLeft: '20px', marginTop: '8px' }}
```
Try to use margin attribute directly margin={{ top: 'medium', left:'large' }} or top:'6px', left:'20px' should work